### PR TITLE
quickstart: Use `PolicyServer` instead of `Policy Server`

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Use `PolicyServer` instead of `Policy Server`, to match the Kubernetes resource name.

### CODE REVIEW

File: src/quick-start.md [hunk 1]
```diff
@@ -47,11 +47,11 @@ Kubewarden has three main components which you will interact with:
 * The ClusterAdmissionPolicy
 * The AdmissionPolicy
 
-### Policy Server
+### PolicyServer
 
-A Kubewarden Policy Server is completely managed by the `kubewarden-controller` and multiple Policy Servers can be deployed in the same Kubernetes cluster.
+A Kubewarden `PolicyServer` is completely managed by the `kubewarden-controller` and multiple `PolicyServers` can be deployed in the same Kubernetes cluster.
```
Comment (divya-mohan0209) [inline src/quick-start.md:52]: I’m not sure we should be capitalising the pluralised version.

Possibly, we can rephrase it to:

```suggestion
A Kubewarden `PolicyServer` is completely managed by the `kubewarden-controller` and multiple instances of the `PolicyServer` can be deployed in the same Kubernetes cluster.
```

**Review by divya-mohan0209 — COMMENTED** [review]
Grammatical nit. Not a blocker, but it would read better.

File: src/quick-start.md (hunk 1 above)
Comment (viccuad) [reply src/quick-start.md:52]: Using the plural is ok, the Kubernetes CRD defines singular and plural names:
https://github.com/kubewarden/kubewarden-controller/blob/e0433fc3774d06dcf5e08bf2c600ad0117b89448/config/crd/bases/policies.kubewarden.io_policyservers.yaml#L15

Comment (divya-mohan0209) [reply src/quick-start.md:52]: Cool, then LGTM! Sorry :)

Comment (viccuad) [reply src/quick-start.md:52]: thanks! merging
